internal/controller/factory/configMap.go

```diff
@@ -74,3 +74,14 @@ func CreateOrUpdateClusterStateConfigMap(
 
 	return reconcileConfigMap(ctx, rclient, cluster.Name, configMap)
 }
+
+// etcdClusterIsReady returns true if condition "Ready" has status equal to "True", otherwise false.
+func etcdClusterIsReady(cluster *etcdaenixiov1alpha1.EtcdCluster) bool {
```

Comment:
I would call it `isEtcdClusterReady`

Reply:
We have function clusterIsReady, that checks amount of ready replicas. And I would say that naming is not clear what is what :) That one function checks condition. Another function checks replicas.

I wild rename this function completely. Like CheckReadyCondition

Reply:
Make sense